Extension/src/LanguageServer/client.ts

```diff
@@ -410,6 +410,10 @@ class DefaultClient implements Client {
                 default: {
                     systemIncludePath: settings.defaultSystemIncludePath
                 },
+                intelliSense: {
```

Comment:
This will also need to change to `intelliSenseCachePath` and `intelliSenseCacheSize`